### PR TITLE
[CSFS-11]  Sign-Up Form Validation

### PR DESCRIPTION
**Pull request by DamisiM**
## Task: [CSFS-11]  Sign-Up Form Validation`
- `DOM task`

### CODE REVIEW

File: submissions/damisiMahmood/domManipulation/signUp.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      "Please enter your mail Address",
+      "Please enter a Valid mail",
+      event);
+    fieldValidation(
+      password,
+      passwordRegex,
+      "Please enter your password",
+      "Your password should hav a minimum of 6 characters, 1 capital letter , 1 special character eg @ and 1 number.",
+      event);
+    fieldValidation(
```
Comment (adaugochi) [inline submissions/damisiMahmood/domManipulation/signUp.js:60]: How does this function checks if `confirmPassword` and `password` matches or not

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — CHANGES_REQUESTED** [review]
@DamisiM you don't have a function that checks if confirmPassword and password matches or not

**Comment by adaugochi** [comment]
What you have done so far is checking if they match a particular regular expression

**Comment by DamisiM** [comment]
Oh, I understand your comment now, fixed

**Review by adaugochi — APPROVED** [review]

**Pull request merged**